index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&display=swap" rel="stylesheet"/>
+
+</head>
+<body>
+    <header class="navbar">
+        <div class="nav-logo">
+            <a class="logo" href="#">My Logo</a>           
+        </div>
+        <nav class="container">
+            <div class="menu-wrapper">
+                <img class="mobile-menu" src="img/menu.png" alt="Navigate">
+            </div>
+        </nav>
+
+    </header>
+     <section class="portfolio"> 
+        <h2 class="intro">I'm Harrison <br> Glad to see you!</h2>   
```

Comment:
- [ ] Good job so far 👏 In addition to what you have done, I would suggest that you use an `<h1>` here instead of h2. This is because it is among the markup guidelines provided and we are expected to implement it. For more info please checkout the following [article](https://www.reliablesoft.net/h1-tag/)😃 👍

Reply:
Rectified